modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+
+.19+|*ap-northeast-2*
+|`ami-0f080e37e37e44f10`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-07cd50c25e097efe5`
+|`rhcos-48.84.202107202156-0`
+|`ami-0144a4f381fc1606c`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0c198ada546e829ef`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0032241003c840b2e`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0c27be2da6c64a816`
+|`rhcos-47.83.202102090044-0`
+|`ami-0b6a0d64e49e51b1c`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-083f8dbc4f43d05c1`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0202e58f8b346aa26`
+|`rhcos-45.81.202005200134-0`
+|`ami-0ec24f2a40bc14964`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0101583602d486d67`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-02cc8dc76de74feed`
+|`rhcos-49.84.202107191624-0`
+|`ami-0d4ceb5e3440900ea`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0e47d438d90b46908`
+|`rhcos-46.82.202008181646-0`
+|`ami-0cb77f66f302754b8`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-087122d87027fc107`
+|`rhcos-48.84.202105062123-0`
+|`ami-05fcd891f761b5a26`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-03d40d15a5d3ba4ff`
+|`rhcos-45.82.202008010929-0`
+|`ami-0198e67ed3464b661`
+|`rhcos-46.82.202008071940-0`
+
+.19+|*ap-southeast-1*
+|`ami-046494e2c14359311`
+|`rhcos-46.82.202008071940-0`
+|`ami-0f4364f94af4aab70`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-00039ae98224ac972`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-032de9543fa88f496`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-025d35766b8bfdd71`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-08b102eea4ca6bd02`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-04843cbe2741982bc`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0c22bbd807a0256d6`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0605f5746fd66f4fb`
+|`rhcos-45.82.202008010929-0`
+|`ami-0eb4e08ce1f20ca2e`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-053bbe72294a548d1`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0eef5030bf996ef45`
+|`rhcos-45.81.202005200134-0`
+|`ami-0e711f6fa6a408d08`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0ada217ae22ba5417`
+|`rhcos-47.83.202102090044-0`
+|`ami-094793700675782d2`
+|`rhcos-48.84.202107202156-0`
+|`ami-016bbb822d84af01d`
+|`rhcos-46.82.202008181646-0`
+|`ami-0f359d62ac28437e2`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f1b225d48a937c73`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0f049ec45fb9e26bd`
+|`rhcos-48.84.202105062123-0`
+
+.19+|*ap-southeast-2*
+|`ami-0c744e459a8289065`
+|`rhcos-47.83.202102090044-0`
+|`ami-05db290d03ae42834`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0c313cb32b39ed6b7`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0503e1dacf597612b`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-065b26ab9484075b2`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-013b5b28f1b3cfc66`
+|`rhcos-46.82.202008181646-0`
+|`ami-0a12387dd4a2d1f10`
+|`rhcos-46.82.202008071940-0`
+|`ami-0fe73f4d11cbecc4d`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?